packages/aragon-wrapper/src/index.js

```diff
@@ -1242,6 +1246,99 @@ export default class Aragon {
     return []
   }
 
+  /**
+   * Calculate the transaction path for a basket of intents.
+   * Expects the `intentBasket` to be an array of tuples holding the following:
+   *   {string}   destination: destination address
+   *   {string}   methodName: method to invoke on destination
+   *   {Array<*>} params: method params
+   * These are the same parameters as the ones used for `getTransactionPath()`
+   *
+   * Allows user to specify how many of the intents should be checked.
+   * `checkMode` supports:
+   *   'all': All intents will be checked to make sure they use the same forwarding path.
+   *   'single': assumes all intents can use the path found from the first intent
+   *
+   * @param  {Array<Array<string, string, Array<*>>>} intentBasket Intents
+   * @param  {Object} [options]
+   * @param  {string} [options.checkMode] Path checking mode
+   * @return {Promise<Array<Object>>} An array of Ethereum transactions that describe each step in the path
+   */
+  async getTransactionPathForIntentBasket (intentBasket, { checkMode = 'all' } = {}) {
+    // Get transaction paths for entire basket
+    const intentsToCheck =
+      checkMode === 'all'
```

Comment:
When would checkMode `all` be used? 

The only usage of it uses `single`:
https://github.com/aragon/aragon/blob/98bf8de6309ad747c7cea1c92e4ffe08823e296c/src/components/Upgrade/UpgradeOrganizationPanel.js#L69-L72

Reply:
Is that because the intentBasket coming from the client will have the same path?

Reply:
`all` isn't used yet, since our only use case is calling the same function, but in the future when you might be including _very different_ transactions (e.g. withdraw from Finance and burn a token at the same time, see https://github.com/aragon/aragon/issues/363), it'll be important to use `all` to check that all the actions can actually be performed together.